.github/workflows/ci.yml

```diff
@@ -8,10 +8,15 @@ on:
 jobs:
   test:
     runs-on: ${{ contains( github.event.pull_request.labels.*.name, 'use_fast_ci') && 'ubuntu-latest-8core' || 'ubuntu-latest' }}
+    strategy:
+      fail-fast: false
```

Comment:
This is `true` by default and would e.g. stop running tests in the event that linting fails. Setting it to `false` will make tests run to completion, even if linting fails.